components/category-toolgroup.vue

```diff
@@ -96,31 +96,29 @@
 
             styleTableRow(p_row) {
                 if (p_row.identifier === this.selectedTool.identifier) {
-                    return "selected-tool";
+                    return "category-style-5";
```

Comment:
I think this works, but we should make an issue to refactor the whole color handling in the store. Because now we're coupled to a specific style name here and in all other places we directly have to read the mapping of categories to styles from the store. 

Would you mind making an issue to go over this whole section here https://github.com/neurobagel/annotation_tool/blob/1161c37e912cfd66fde376e1a6c8755c87a223bd/store/index.js#L62-L91 and discussing how to make it easier to use?

Reply:
Done #631